tests/installation/partitioning_raid.pm

```diff
@@ -196,32 +207,56 @@ sub run() {
 
         # in last step of for loop edit first vda1 and format it as EFI ESP, preparation for fate#322485
         if ($_ == 4 and get_var('UEFI')) {
+            wait_idle 3;
```

Comment:
please don't, see https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/3050#discussion_r121356102
Maybe you are misunderstanding what `wait_idle` is actually doing. If you just want to use it for making sure the right screenshot is recorded instead of using interactive mode either keep it for now if it does the job or replace it by sleep which is in this case probably more resembling what you really need.